WebRoot/js/zimbraMail/briefcase/controller/ZmBriefcaseController.js

```diff
@@ -400,6 +409,47 @@ function(parent, num) {
     }
 };
 
+// Show / hide file toolbar and context menu options.
+ZmBriefcaseController.prototype.setFileOperations =
+function(parent, show) {
+	var fileOperations = [
+		ZmOperation.OPEN_FILE,
```

Comment:
move this to zimlet

Reply:
Done. Moved OPEN_FILE operation to zimlet code.